hr_salary_rule_reference/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2013 Savoir-faire Linux
```

Comment:
same.

Reply:
I did not do any customizations on this file, so i think it's good to keep it like that